frontend/src/components/admin/userManagement/UserAddModify.js

```diff
@@ -1031,15 +1033,15 @@ function UserAddModify() {
                     </>
                   </Column>
                   <Column lg={8} md={4} sm={4}>
-                    <TextInput
+                    <input
```

Comment:
we are using Carbon as a Style Guide so we dont encourage using the default html components